pandas/core/strings.py

```diff
@@ -2151,12 +2151,60 @@ def encode(self, encoding, errors="strict"):
         return self._wrap_result(result)
 
     _shared_docs['str_strip'] = ("""
-    Strip whitespace (including newlines) from each string in the
-    Series/Index from %(side)s. Equivalent to :meth:`str.%(method)s`.
+    Remove leading and trailing characters.
+
+    Strip whitespaces (including newlines) or a set of specified characters
+    from each string in the Series/Index from %(side)s.
+    Equivalent to :meth:`str.%(method)s`.
+
+    Parameters
+    ----------
+    to_strip : str or None, default None (whitespaces are removed)
```

Comment:
the coment about whitespaces are removed should be on the next line with a case, e.g.
If None then whitespaces are removed